vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
+
+| TYPE | REQUIRED | PLATFORM |
+|:--:|:--|:--|
+| boolean | Yes | Windows, macOS |
+
+[React Native Web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-9fe3ab194c72e9837a8ffc3e7c44f33d) introduced  `hovered` on `Pressable` to enable common mouse scenarios. In order to align and make this complete the work is as follows:
+
+- Windows has added `hovered` and marked `onMouseEnter/Leave` as to be deprecated [issue TBD]()
+- Add `hovered` to `Pressable` [issue TBD]()
+- macOS has added 'hovered' and marked their View component implementation of `onMouseEnter/Leave` as deprecated [issue TBD]()
+
+### `hovered` example
+
+``` js
+<Pressable style={({ hovered, pressed, focused }) => {
+          console.log(focused);
+          let backgroundColor = 'white';
+          if (hovered) {
+            backgroundColor = 'lightgray';
+          }
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
+The following callbacks are available on `View` component (and get passed through to Pressable) in Windows and macOS to cover the most common use cases where mouse handling would occur.
```

Comment:
Interested in the part about passing through to `Pressable`, and sort of how that reconciles with hover detection. Hover detection right now has some more components than when the mouse is entered, (e.g. it's time sensitive), so `onPointerOver` wouldn't neccesarily be an analog of `onHoverOver`. Wondering if we should try to expose the higher level hover related events instead of routing through raw pointer over separately.